2025-08-26 13:03:42.949 +03:00 [FTL] Application startup exception
System.InvalidOperationException: Unable to resolve service for type 'TaskTrackerClean.Application.Services.RabbitMQProducer' while attempting to activate 'TaskTrackerClean.API.Middlewares.ExceptionHandlingMiddleware'.
   at Microsoft.Extensions.Internal.ActivatorUtilities.ConstructorMatcher.CreateInstance(IServiceProvider provider)
   at Microsoft.Extensions.Internal.ActivatorUtilities.CreateInstance(IServiceProvider provider, Type instanceType, Object[] parameters)
   at Microsoft.AspNetCore.Builder.UseMiddlewareExtensions.ReflectionMiddlewareBinder.CreateMiddleware(RequestDelegate next)
   at Microsoft.AspNetCore.Builder.ApplicationBuilder.Build()
   at Microsoft.AspNetCore.Builder.ApplicationBuilder.Build()
   at Microsoft.AspNetCore.Hosting.GenericWebHostService.StartAsync(CancellationToken cancellationToken)
2025-08-26 13:03:43.330 +03:00 [ERR] Hosting failed to start
System.InvalidOperationException: Unable to resolve service for type 'TaskTrackerClean.Application.Services.RabbitMQProducer' while attempting to activate 'TaskTrackerClean.API.Middlewares.ExceptionHandlingMiddleware'.
   at Microsoft.Extensions.Internal.ActivatorUtilities.ConstructorMatcher.CreateInstance(IServiceProvider provider)
   at Microsoft.Extensions.Internal.ActivatorUtilities.CreateInstance(IServiceProvider provider, Type instanceType, Object[] parameters)
   at Microsoft.AspNetCore.Builder.UseMiddlewareExtensions.ReflectionMiddlewareBinder.CreateMiddleware(RequestDelegate next)
   at Microsoft.AspNetCore.Builder.ApplicationBuilder.Build()
   at Microsoft.AspNetCore.Builder.ApplicationBuilder.Build()
   at Microsoft.AspNetCore.Hosting.GenericWebHostService.StartAsync(CancellationToken cancellationToken)
   at Microsoft.Extensions.Hosting.Internal.Host.<StartAsync>b__14_1(IHostedService service, CancellationToken token)
   at Microsoft.Extensions.Hosting.Internal.Host.ForeachService[T](IEnumerable`1 services, CancellationToken token, Boolean concurrent, Boolean abortOnFirstException, List`1 exceptions, Func`3 operation)
2025-08-26 13:06:17.637 +03:00 [FTL] Application startup exception
System.InvalidOperationException: Cannot resolve scoped service 'TaskTrackerClean.Application.Services.RabbitMQProducer' from root provider.
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteValidator.ValidateResolution(ServiceCallSite callSite, IServiceScope scope, IServiceScope rootScope)
   at Microsoft.Extensions.DependencyInjection.ServiceProvider.OnResolve(ServiceCallSite callSite, IServiceScope scope)
   at Microsoft.Extensions.DependencyInjection.ServiceProvider.GetService(ServiceIdentifier serviceIdentifier, ServiceProviderEngineScope serviceProviderEngineScope)
   at Microsoft.Extensions.DependencyInjection.ServiceProvider.GetService(Type serviceType)
   at Microsoft.Extensions.Internal.ActivatorUtilities.ConstructorMatcher.CreateInstance(IServiceProvider provider)
   at Microsoft.Extensions.Internal.ActivatorUtilities.CreateInstance(IServiceProvider provider, Type instanceType, Object[] parameters)
   at Microsoft.AspNetCore.Builder.UseMiddlewareExtensions.ReflectionMiddlewareBinder.CreateMiddleware(RequestDelegate next)
   at Microsoft.AspNetCore.Builder.ApplicationBuilder.Build()
   at Microsoft.AspNetCore.Builder.ApplicationBuilder.Build()
   at Microsoft.AspNetCore.Hosting.GenericWebHostService.StartAsync(CancellationToken cancellationToken)
2025-08-26 13:06:17.772 +03:00 [ERR] Hosting failed to start
System.InvalidOperationException: Cannot resolve scoped service 'TaskTrackerClean.Application.Services.RabbitMQProducer' from root provider.
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteValidator.ValidateResolution(ServiceCallSite callSite, IServiceScope scope, IServiceScope rootScope)
   at Microsoft.Extensions.DependencyInjection.ServiceProvider.OnResolve(ServiceCallSite callSite, IServiceScope scope)
   at Microsoft.Extensions.DependencyInjection.ServiceProvider.GetService(ServiceIdentifier serviceIdentifier, ServiceProviderEngineScope serviceProviderEngineScope)
   at Microsoft.Extensions.DependencyInjection.ServiceProvider.GetService(Type serviceType)
   at Microsoft.Extensions.Internal.ActivatorUtilities.ConstructorMatcher.CreateInstance(IServiceProvider provider)
   at Microsoft.Extensions.Internal.ActivatorUtilities.CreateInstance(IServiceProvider provider, Type instanceType, Object[] parameters)
   at Microsoft.AspNetCore.Builder.UseMiddlewareExtensions.ReflectionMiddlewareBinder.CreateMiddleware(RequestDelegate next)
   at Microsoft.AspNetCore.Builder.ApplicationBuilder.Build()
   at Microsoft.AspNetCore.Builder.ApplicationBuilder.Build()
   at Microsoft.AspNetCore.Hosting.GenericWebHostService.StartAsync(CancellationToken cancellationToken)
   at Microsoft.Extensions.Hosting.Internal.Host.<StartAsync>b__14_1(IHostedService service, CancellationToken token)
   at Microsoft.Extensions.Hosting.Internal.Host.ForeachService[T](IEnumerable`1 services, CancellationToken token, Boolean concurrent, Boolean abortOnFirstException, List`1 exceptions, Func`3 operation)
2025-08-26 13:20:33.190 +03:00 [ERR] An error has occurred. 
traceId: 0HNF4H8L56HTA:00000003 

System.InvalidOperationException: Project with this title already exists
   at TaskTrackerClean.Application.Services.ProjectService.CreateAsync(CreateProjectDto dto, String createdBy) in C:\Users\Sedra\Documents\My .Net Projects\TaskTrackerClean\src\Microservices\TaskTrackerCleanService\TaskTrackerClean.Application\Services\ProjectService.cs:line 27
   at TaskTrackerClean.API.Controllers.ProjectsController.Create(CreateProjectDto dto) in C:\Users\Sedra\Documents\My .Net Projects\TaskTrackerClean\src\Microservices\TaskTrackerCleanService\TaskTrackerClean.API\Controllers\ProjectsController.cs:line 24
   at lambda_method8(Closure, Object)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.AwaitableObjectResultExecutor.Execute(ActionContext actionContext, IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeInnerFilterAsync>g__Awaited|13_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|20_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at TaskTrackerClean.API.Middlewares.ExceptionHandlingMiddleware.Invoke(HttpContext context) in C:\Users\Sedra\Documents\My .Net Projects\TaskTrackerClean\src\Microservices\TaskTrackerCleanService\TaskTrackerClean.API\Middlewares\ExceptionHandlingMiddleware.cs:line 32
2025-08-26 13:22:32.464 +03:00 [ERR] An error has occurred. 
traceId: 0HNF4H8L56HTA:00000004 

System.InvalidOperationException: Project with this title already exists
   at TaskTrackerClean.Application.Services.ProjectService.CreateAsync(CreateProjectDto dto, String createdBy) in C:\Users\Sedra\Documents\My .Net Projects\TaskTrackerClean\src\Microservices\TaskTrackerCleanService\TaskTrackerClean.Application\Services\ProjectService.cs:line 27
   at TaskTrackerClean.API.Controllers.ProjectsController.Create(CreateProjectDto dto) in C:\Users\Sedra\Documents\My .Net Projects\TaskTrackerClean\src\Microservices\TaskTrackerCleanService\TaskTrackerClean.API\Controllers\ProjectsController.cs:line 24
   at lambda_method8(Closure, Object)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.AwaitableObjectResultExecutor.Execute(ActionContext actionContext, IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeInnerFilterAsync>g__Awaited|13_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|20_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at TaskTrackerClean.API.Middlewares.ExceptionHandlingMiddleware.Invoke(HttpContext context) in C:\Users\Sedra\Documents\My .Net Projects\TaskTrackerClean\src\Microservices\TaskTrackerCleanService\TaskTrackerClean.API\Middlewares\ExceptionHandlingMiddleware.cs:line 32
2025-08-26 13:22:42.651 +03:00 [ERR] An error has occurred. 
traceId: 0HNF4H8L56HTA:00000005 

System.InvalidOperationException: Project with this title already exists
   at TaskTrackerClean.Application.Services.ProjectService.CreateAsync(CreateProjectDto dto, String createdBy) in C:\Users\Sedra\Documents\My .Net Projects\TaskTrackerClean\src\Microservices\TaskTrackerCleanService\TaskTrackerClean.Application\Services\ProjectService.cs:line 27
   at TaskTrackerClean.API.Controllers.ProjectsController.Create(CreateProjectDto dto) in C:\Users\Sedra\Documents\My .Net Projects\TaskTrackerClean\src\Microservices\TaskTrackerCleanService\TaskTrackerClean.API\Controllers\ProjectsController.cs:line 24
   at lambda_method8(Closure, Object)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.AwaitableObjectResultExecutor.Execute(ActionContext actionContext, IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeInnerFilterAsync>g__Awaited|13_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|20_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at TaskTrackerClean.API.Middlewares.ExceptionHandlingMiddleware.Invoke(HttpContext context) in C:\Users\Sedra\Documents\My .Net Projects\TaskTrackerClean\src\Microservices\TaskTrackerCleanService\TaskTrackerClean.API\Middlewares\ExceptionHandlingMiddleware.cs:line 32
2025-08-26 13:26:50.227 +03:00 [ERR] An error has occurred. 
traceId: 0HNF4HC3NNKG0:00000003 

System.InvalidOperationException: Project with this title already exists
   at TaskTrackerClean.Application.Services.ProjectService.CreateAsync(CreateProjectDto dto, String createdBy) in C:\Users\Sedra\Documents\My .Net Projects\TaskTrackerClean\src\Microservices\TaskTrackerCleanService\TaskTrackerClean.Application\Services\ProjectService.cs:line 27
   at TaskTrackerClean.API.Controllers.ProjectsController.Create(CreateProjectDto dto) in C:\Users\Sedra\Documents\My .Net Projects\TaskTrackerClean\src\Microservices\TaskTrackerCleanService\TaskTrackerClean.API\Controllers\ProjectsController.cs:line 24
   at lambda_method8(Closure, Object)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.AwaitableObjectResultExecutor.Execute(ActionContext actionContext, IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeInnerFilterAsync>g__Awaited|13_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|20_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at TaskTrackerClean.API.Middlewares.ExceptionHandlingMiddleware.Invoke(HttpContext context) in C:\Users\Sedra\Documents\My .Net Projects\TaskTrackerClean\src\Microservices\TaskTrackerCleanService\TaskTrackerClean.API\Middlewares\ExceptionHandlingMiddleware.cs:line 32
2025-08-26 13:33:12.412 +03:00 [ERR] An error has occurred. 
traceId: 0HNF4HFJV04HD:00000003 

System.InvalidOperationException: Project with this title already exists
   at TaskTrackerClean.Application.Services.ProjectService.CreateAsync(CreateProjectDto dto, String createdBy) in C:\Users\Sedra\Documents\My .Net Projects\TaskTrackerClean\src\Microservices\TaskTrackerCleanService\TaskTrackerClean.Application\Services\ProjectService.cs:line 27
   at TaskTrackerClean.API.Controllers.ProjectsController.Create(CreateProjectDto dto) in C:\Users\Sedra\Documents\My .Net Projects\TaskTrackerClean\src\Microservices\TaskTrackerCleanService\TaskTrackerClean.API\Controllers\ProjectsController.cs:line 24
   at lambda_method8(Closure, Object)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.AwaitableObjectResultExecutor.Execute(ActionContext actionContext, IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeInnerFilterAsync>g__Awaited|13_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|20_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at TaskTrackerClean.API.Middlewares.ExceptionHandlingMiddleware.Invoke(HttpContext context) in C:\Users\Sedra\Documents\My .Net Projects\TaskTrackerClean\src\Microservices\TaskTrackerCleanService\TaskTrackerClean.API\Middlewares\ExceptionHandlingMiddleware.cs:line 32
2025-08-26 13:33:12.825 +03:00 [ERR] An unhandled exception has occurred while executing the request.
RabbitMQ.Client.Exceptions.OperationInterruptedException: The AMQP operation was interrupted: AMQP close-reason, initiated by Peer, code=404, text='NOT_FOUND - no queue 'exception.logs' in vhost 'loggerhost'', classId=50, methodId=20
   at RabbitMQ.Client.Impl.Channel.QueueBindAsync(String queue, String exchange, String routingKey, IDictionary`2 arguments, Boolean noWait, CancellationToken cancellationToken)
   at RabbitMQ.Client.Impl.AutorecoveringChannel.QueueBindAsync(String queue, String exchange, String routingKey, IDictionary`2 arguments, Boolean noWait, CancellationToken cancellationToken)
   at TaskTrackerClean.Application.Services.RabbitMQProducer.EnsureConnectedAsync() in C:\Users\Sedra\Documents\My .Net Projects\TaskTrackerClean\src\Microservices\TaskTrackerCleanService\TaskTrackerClean.Application\Services\RabbitMQProducer.cs:line 51
   at TaskTrackerClean.Application.Services.RabbitMQProducer.PublishLog(Object body, String queueName) in C:\Users\Sedra\Documents\My .Net Projects\TaskTrackerClean\src\Microservices\TaskTrackerCleanService\TaskTrackerClean.Application\Services\RabbitMQProducer.cs:line 59
   at TaskTrackerClean.API.Middlewares.ExceptionHandlingMiddleware.Invoke(HttpContext context) in C:\Users\Sedra\Documents\My .Net Projects\TaskTrackerClean\src\Microservices\TaskTrackerCleanService\TaskTrackerClean.API\Middlewares\ExceptionHandlingMiddleware.cs:line 48
   at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)
   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddlewareImpl.Invoke(HttpContext context)
2025-08-26 13:38:03.120 +03:00 [ERR] An error has occurred. 
traceId: 0HNF4HID38CJF:00000003 

System.InvalidOperationException: Project with this title already exists
   at TaskTrackerClean.Application.Services.ProjectService.CreateAsync(CreateProjectDto dto, String createdBy) in C:\Users\Sedra\Documents\My .Net Projects\TaskTrackerClean\src\Microservices\TaskTrackerCleanService\TaskTrackerClean.Application\Services\ProjectService.cs:line 27
   at TaskTrackerClean.API.Controllers.ProjectsController.Create(CreateProjectDto dto) in C:\Users\Sedra\Documents\My .Net Projects\TaskTrackerClean\src\Microservices\TaskTrackerCleanService\TaskTrackerClean.API\Controllers\ProjectsController.cs:line 24
   at lambda_method8(Closure, Object)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.AwaitableObjectResultExecutor.Execute(ActionContext actionContext, IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeInnerFilterAsync>g__Awaited|13_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|20_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at TaskTrackerClean.API.Middlewares.ExceptionHandlingMiddleware.Invoke(HttpContext context) in C:\Users\Sedra\Documents\My .Net Projects\TaskTrackerClean\src\Microservices\TaskTrackerCleanService\TaskTrackerClean.API\Middlewares\ExceptionHandlingMiddleware.cs:line 32
2025-08-26 13:38:40.742 +03:00 [ERR] An error has occurred. 
traceId: 0HNF4HID38CJF:00000004 

System.InvalidOperationException: Project with this title already exists
   at TaskTrackerClean.Application.Services.ProjectService.CreateAsync(CreateProjectDto dto, String createdBy) in C:\Users\Sedra\Documents\My .Net Projects\TaskTrackerClean\src\Microservices\TaskTrackerCleanService\TaskTrackerClean.Application\Services\ProjectService.cs:line 27
   at TaskTrackerClean.API.Controllers.ProjectsController.Create(CreateProjectDto dto) in C:\Users\Sedra\Documents\My .Net Projects\TaskTrackerClean\src\Microservices\TaskTrackerCleanService\TaskTrackerClean.API\Controllers\ProjectsController.cs:line 24
   at lambda_method8(Closure, Object)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.AwaitableObjectResultExecutor.Execute(ActionContext actionContext, IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeInnerFilterAsync>g__Awaited|13_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|20_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at TaskTrackerClean.API.Middlewares.ExceptionHandlingMiddleware.Invoke(HttpContext context) in C:\Users\Sedra\Documents\My .Net Projects\TaskTrackerClean\src\Microservices\TaskTrackerCleanService\TaskTrackerClean.API\Middlewares\ExceptionHandlingMiddleware.cs:line 32
2025-08-26 13:38:49.758 +03:00 [ERR] An error has occurred. 
traceId: 0HNF4HID38CJF:00000005 

System.InvalidOperationException: Project with this title already exists
   at TaskTrackerClean.Application.Services.ProjectService.CreateAsync(CreateProjectDto dto, String createdBy) in C:\Users\Sedra\Documents\My .Net Projects\TaskTrackerClean\src\Microservices\TaskTrackerCleanService\TaskTrackerClean.Application\Services\ProjectService.cs:line 27
   at TaskTrackerClean.API.Controllers.ProjectsController.Create(CreateProjectDto dto) in C:\Users\Sedra\Documents\My .Net Projects\TaskTrackerClean\src\Microservices\TaskTrackerCleanService\TaskTrackerClean.API\Controllers\ProjectsController.cs:line 24
   at lambda_method8(Closure, Object)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.AwaitableObjectResultExecutor.Execute(ActionContext actionContext, IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeInnerFilterAsync>g__Awaited|13_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|20_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at TaskTrackerClean.API.Middlewares.ExceptionHandlingMiddleware.Invoke(HttpContext context) in C:\Users\Sedra\Documents\My .Net Projects\TaskTrackerClean\src\Microservices\TaskTrackerCleanService\TaskTrackerClean.API\Middlewares\ExceptionHandlingMiddleware.cs:line 32
2025-08-26 13:54:14.003 +03:00 [ERR] An error has occurred. 
traceId: 0HNF4HQRHIST7:00000003 

System.InvalidOperationException: Project with this title already exists
   at TaskTrackerClean.Application.Services.ProjectService.CreateAsync(CreateProjectDto dto, String createdBy) in C:\Users\Sedra\Documents\My .Net Projects\TaskTrackerClean\src\Microservices\TaskTrackerCleanService\TaskTrackerClean.Application\Services\ProjectService.cs:line 27
   at TaskTrackerClean.API.Controllers.ProjectsController.Create(CreateProjectDto dto) in C:\Users\Sedra\Documents\My .Net Projects\TaskTrackerClean\src\Microservices\TaskTrackerCleanService\TaskTrackerClean.API\Controllers\ProjectsController.cs:line 24
   at lambda_method8(Closure, Object)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.AwaitableObjectResultExecutor.Execute(ActionContext actionContext, IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeInnerFilterAsync>g__Awaited|13_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|20_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at TaskTrackerClean.API.Middlewares.ExceptionHandlingMiddleware.Invoke(HttpContext context) in C:\Users\Sedra\Documents\My .Net Projects\TaskTrackerClean\src\Microservices\TaskTrackerCleanService\TaskTrackerClean.API\Middlewares\ExceptionHandlingMiddleware.cs:line 32
2025-08-26 14:23:23.449 +03:00 [ERR] An error has occurred. 
traceId: 0HNF4IBJN3ECL:00000003 

System.InvalidOperationException: Project with this title already exists
   at TaskTrackerClean.Application.Services.ProjectService.CreateAsync(CreateProjectDto dto, String createdBy) in C:\Users\Sedra\Documents\My .Net Projects\TaskTrackerClean\src\Microservices\TaskTrackerCleanService\TaskTrackerClean.Application\Services\ProjectService.cs:line 27
   at TaskTrackerClean.API.Controllers.ProjectsController.Create(CreateProjectDto dto) in C:\Users\Sedra\Documents\My .Net Projects\TaskTrackerClean\src\Microservices\TaskTrackerCleanService\TaskTrackerClean.API\Controllers\ProjectsController.cs:line 24
   at lambda_method8(Closure, Object)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.AwaitableObjectResultExecutor.Execute(ActionContext actionContext, IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeInnerFilterAsync>g__Awaited|13_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|20_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at TaskTrackerClean.API.Middlewares.ExceptionHandlingMiddleware.Invoke(HttpContext context) in C:\Users\Sedra\Documents\My .Net Projects\TaskTrackerClean\src\Microservices\TaskTrackerCleanService\TaskTrackerClean.API\Middlewares\ExceptionHandlingMiddleware.cs:line 32
2025-08-26 14:29:50.134 +03:00 [ERR] An error has occurred. 
traceId: 0HNF4IF9CERH8:00000003 

System.InvalidOperationException: Project with this title already exists
   at TaskTrackerClean.Application.Services.ProjectService.CreateAsync(CreateProjectDto dto, String createdBy) in C:\Users\Sedra\Documents\My .Net Projects\TaskTrackerClean\src\Microservices\TaskTrackerCleanService\TaskTrackerClean.Application\Services\ProjectService.cs:line 27
   at TaskTrackerClean.API.Controllers.ProjectsController.Create(CreateProjectDto dto) in C:\Users\Sedra\Documents\My .Net Projects\TaskTrackerClean\src\Microservices\TaskTrackerCleanService\TaskTrackerClean.API\Controllers\ProjectsController.cs:line 24
   at lambda_method8(Closure, Object)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.AwaitableObjectResultExecutor.Execute(ActionContext actionContext, IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeInnerFilterAsync>g__Awaited|13_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|20_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at TaskTrackerClean.API.Middlewares.ExceptionHandlingMiddleware.Invoke(HttpContext context) in C:\Users\Sedra\Documents\My .Net Projects\TaskTrackerClean\src\Microservices\TaskTrackerCleanService\TaskTrackerClean.API\Middlewares\ExceptionHandlingMiddleware.cs:line 32
2025-08-26 17:13:22.138 +03:00 [FTL] Application startup exception
System.InvalidOperationException: Cannot resolve scoped service 'MassTransit.IPublishEndpoint' from root provider.
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteValidator.ValidateResolution(ServiceCallSite callSite, IServiceScope scope, IServiceScope rootScope)
   at Microsoft.Extensions.DependencyInjection.ServiceProvider.OnResolve(ServiceCallSite callSite, IServiceScope scope)
   at Microsoft.Extensions.DependencyInjection.ServiceProvider.GetService(ServiceIdentifier serviceIdentifier, ServiceProviderEngineScope serviceProviderEngineScope)
   at Microsoft.Extensions.DependencyInjection.ServiceProvider.GetService(Type serviceType)
   at Microsoft.Extensions.Internal.ActivatorUtilities.ConstructorMatcher.CreateInstance(IServiceProvider provider)
   at Microsoft.Extensions.Internal.ActivatorUtilities.CreateInstance(IServiceProvider provider, Type instanceType, Object[] parameters)
   at Microsoft.AspNetCore.Builder.UseMiddlewareExtensions.ReflectionMiddlewareBinder.CreateMiddleware(RequestDelegate next)
   at Microsoft.AspNetCore.Builder.ApplicationBuilder.Build()
   at Microsoft.AspNetCore.Builder.ApplicationBuilder.Build()
   at Microsoft.AspNetCore.Hosting.GenericWebHostService.StartAsync(CancellationToken cancellationToken)
2025-08-26 17:13:22.257 +03:00 [ERR] Hosting failed to start
System.InvalidOperationException: Cannot resolve scoped service 'MassTransit.IPublishEndpoint' from root provider.
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteValidator.ValidateResolution(ServiceCallSite callSite, IServiceScope scope, IServiceScope rootScope)
   at Microsoft.Extensions.DependencyInjection.ServiceProvider.OnResolve(ServiceCallSite callSite, IServiceScope scope)
   at Microsoft.Extensions.DependencyInjection.ServiceProvider.GetService(ServiceIdentifier serviceIdentifier, ServiceProviderEngineScope serviceProviderEngineScope)
   at Microsoft.Extensions.DependencyInjection.ServiceProvider.GetService(Type serviceType)
   at Microsoft.Extensions.Internal.ActivatorUtilities.ConstructorMatcher.CreateInstance(IServiceProvider provider)
   at Microsoft.Extensions.Internal.ActivatorUtilities.CreateInstance(IServiceProvider provider, Type instanceType, Object[] parameters)
   at Microsoft.AspNetCore.Builder.UseMiddlewareExtensions.ReflectionMiddlewareBinder.CreateMiddleware(RequestDelegate next)
   at Microsoft.AspNetCore.Builder.ApplicationBuilder.Build()
   at Microsoft.AspNetCore.Builder.ApplicationBuilder.Build()
   at Microsoft.AspNetCore.Hosting.GenericWebHostService.StartAsync(CancellationToken cancellationToken)
   at Microsoft.Extensions.Hosting.Internal.Host.<StartAsync>b__14_1(IHostedService service, CancellationToken token)
   at Microsoft.Extensions.Hosting.Internal.Host.ForeachService[T](IEnumerable`1 services, CancellationToken token, Boolean concurrent, Boolean abortOnFirstException, List`1 exceptions, Func`3 operation)
2025-08-26 17:19:34.417 +03:00 [FTL] Application startup exception
System.InvalidOperationException: Cannot resolve scoped service 'MassTransit.IPublishEndpoint' from root provider.
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteValidator.ValidateResolution(ServiceCallSite callSite, IServiceScope scope, IServiceScope rootScope)
   at Microsoft.Extensions.DependencyInjection.ServiceProvider.OnResolve(ServiceCallSite callSite, IServiceScope scope)
   at Microsoft.Extensions.DependencyInjection.ServiceProvider.GetService(ServiceIdentifier serviceIdentifier, ServiceProviderEngineScope serviceProviderEngineScope)
   at Microsoft.Extensions.DependencyInjection.ServiceProvider.GetService(Type serviceType)
   at Microsoft.Extensions.Internal.ActivatorUtilities.ConstructorMatcher.CreateInstance(IServiceProvider provider)
   at Microsoft.Extensions.Internal.ActivatorUtilities.CreateInstance(IServiceProvider provider, Type instanceType, Object[] parameters)
   at Microsoft.AspNetCore.Builder.UseMiddlewareExtensions.ReflectionMiddlewareBinder.CreateMiddleware(RequestDelegate next)
   at Microsoft.AspNetCore.Builder.ApplicationBuilder.Build()
   at Microsoft.AspNetCore.Builder.ApplicationBuilder.Build()
   at Microsoft.AspNetCore.Hosting.GenericWebHostService.StartAsync(CancellationToken cancellationToken)
2025-08-26 17:19:34.516 +03:00 [ERR] Hosting failed to start
System.InvalidOperationException: Cannot resolve scoped service 'MassTransit.IPublishEndpoint' from root provider.
   at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteValidator.ValidateResolution(ServiceCallSite callSite, IServiceScope scope, IServiceScope rootScope)
   at Microsoft.Extensions.DependencyInjection.ServiceProvider.OnResolve(ServiceCallSite callSite, IServiceScope scope)
   at Microsoft.Extensions.DependencyInjection.ServiceProvider.GetService(ServiceIdentifier serviceIdentifier, ServiceProviderEngineScope serviceProviderEngineScope)
   at Microsoft.Extensions.DependencyInjection.ServiceProvider.GetService(Type serviceType)
   at Microsoft.Extensions.Internal.ActivatorUtilities.ConstructorMatcher.CreateInstance(IServiceProvider provider)
   at Microsoft.Extensions.Internal.ActivatorUtilities.CreateInstance(IServiceProvider provider, Type instanceType, Object[] parameters)
   at Microsoft.AspNetCore.Builder.UseMiddlewareExtensions.ReflectionMiddlewareBinder.CreateMiddleware(RequestDelegate next)
   at Microsoft.AspNetCore.Builder.ApplicationBuilder.Build()
   at Microsoft.AspNetCore.Builder.ApplicationBuilder.Build()
   at Microsoft.AspNetCore.Hosting.GenericWebHostService.StartAsync(CancellationToken cancellationToken)
   at Microsoft.Extensions.Hosting.Internal.Host.<StartAsync>b__14_1(IHostedService service, CancellationToken token)
   at Microsoft.Extensions.Hosting.Internal.Host.ForeachService[T](IEnumerable`1 services, CancellationToken token, Boolean concurrent, Boolean abortOnFirstException, List`1 exceptions, Func`3 operation)
2025-08-26 17:27:28.200 +03:00 [ERR] Unhandled exception occurred
System.InvalidOperationException: Project with this title already exists
   at TaskTrackerClean.Application.Services.ProjectService.CreateAsync(CreateProjectDto dto, String createdBy) in C:\Users\Sedra\Documents\My .Net Projects\TaskTrackerClean\src\Microservices\TaskTrackerCleanService\TaskTrackerClean.Application\Services\ProjectService.cs:line 27
   at TaskTrackerClean.API.Controllers.ProjectsController.Create(CreateProjectDto dto) in C:\Users\Sedra\Documents\My .Net Projects\TaskTrackerClean\src\Microservices\TaskTrackerCleanService\TaskTrackerClean.API\Controllers\ProjectsController.cs:line 24
   at lambda_method8(Closure, Object)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.AwaitableObjectResultExecutor.Execute(ActionContext actionContext, IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeInnerFilterAsync>g__Awaited|13_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|20_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at TaskTrackerClean.API.Middlewares.ExceptionHandlingMiddleware.InvokeAsync(HttpContext context) in C:\Users\Sedra\Documents\My .Net Projects\TaskTrackerClean\src\Microservices\TaskTrackerCleanService\TaskTrackerClean.API\Middlewares\ExceptionHandlingMiddleware.cs:line 22
